CHANGELOG.md

```diff
@@ -5,7 +5,97 @@ All notable changes to this project will be documented in this file.
+- Add /internal/translate/keys endpoint ([#1751](https://github.com/pilosa/pilosa/pull/1751))
+- CircleCI: Add race detector to parallel build, default to Go 1.11. ([#1756](https://github.com/pilosa/pilosa/pull/1756))
+- Add distributed tracing. ([#1684](https://github.com/pilosa/pilosa/pull/1684))
+- Add NoStandardView field option ([#1733](https://github.com/pilosa/pilosa/pull/1733))
+- Import roaring endpoint accepts a list of views ([#1738](https://github.com/pilosa/pilosa/pull/1738))
+- Add some stat tracking to roaring implementation ([#1743](https://github.com/pilosa/pilosa/pull/1743))
+- Cluster tests ([#1717](https://github.com/pilosa/pilosa/pull/1717))
+- Allow backslash, carriage return in PQL strings ([#1713](https://github.com/pilosa/pilosa/pull/1713))
+- Add base system, curl and jq for debug and checks ([#1707](https://github.com/pilosa/pilosa/pull/1707))
+- Add "Rows" and "GroupBy" functionality ([#1647](https://github.com/pilosa/pilosa/pull/1647))
+- Add `clear` functional option for imports ([#1699](https://github.com/pilosa/pilosa/pull/1699))
+- Implement tracking of available shards to help support sparse datasets ([#1600](https://github.com/pilosa/pilosa/pull/1600), [#1695](https://github.com/pilosa/pilosa/pull/1695), [#1624](https://github.com/pilosa/pilosa/pull/1624), [#1663](https://github.com/pilosa/pilosa/pull/1663))
+- Add missing rowID/Key columnID/Key tests ([#1683](https://github.com/pilosa/pilosa/pull/1683))
+- Add Store() operation to PQL ([#1666](https://github.com/pilosa/pilosa/pull/1666))
+- Add diagnostics CPUArch field ([#1671](https://github.com/pilosa/pilosa/pull/1671))
+- Add CircleCI step to generate Docker image and push to Docker hub ([#1673](https://github.com/pilosa/pilosa/pull/1673))
+- Implement ClearRow() query ([#1645](https://github.com/pilosa/pilosa/pull/1645))
+- Add support for Bool fields ([#1658](https://github.com/pilosa/pilosa/pull/1658))
+- Make translate map size configurable ([#1653](https://github.com/pilosa/pilosa/pull/1653))
+- Add DirectAdd function to roaring.Bitmap ([#1646](https://github.com/pilosa/pilosa/pull/1646))
+- Roaring import ([#1622](https://github.com/pilosa/pilosa/pull/1622))
+- Add Not() query ([#1635](https://github.com/pilosa/pilosa/pull/1635))
+- Implement Options call and excludeRowAttrs, excludeColumns, columnAttrs and shards args ([#1631](https://github.com/pilosa/pilosa/pull/1631))
+- Add field options to pilosa import ([#1625](https://github.com/pilosa/pilosa/pull/1625))
+- Implement column existence tracking ([#1788](https://github.com/pilosa/pilosa/pull/1788), [#1672](https://github.com/pilosa/pilosa/pull/1672), [#1628](https://github.com/pilosa/pilosa/pull/1628))
+
+### Changed
+
+- Convert the anti-entropy logic to use `ImportRoaring` instead of `QueryNode` ([#1780](https://github.com/pilosa/pilosa/pull/1780))
+- Simplify "require-*" logic in Makefile ([#1755](https://github.com/pilosa/pilosa/pull/1755))
```

Comment:
Is this asterisk going to cause problems in formatting later (I only ask because it does here in this issue... it's making everything that follows italicized).